### PR TITLE
#CNV-35102: new disaster recovery section

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.15+

Issue: [CNV-35102](https://issues.redhat.com//browse/CNV-35102)

Link to docs preview: https://70454--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/backup_restore/virt-disaster-recovery 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by dfitzmau** [comment]
/remove-label peer-review-in-progress

/label peer-review-done

Hi @sabrinajess . Nice work on the PR. LGTM. The embedded link for the [Metropolitan Disaster Recovery (Metro-DR)](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.14/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#metro-dr-solution) heading might be missed by screen readers, so consider adding a link to the end of the section.

File: virt/backup_restore/virt-disaster-recovery.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+:context: virt-about-dr
+
+toc::[]
+
+The disaster recovery documentation provides information for administrators on
+how to recover from certain disaster scenarios that might occur with an
+{product-title} cluster. As an administrator, you must plan your {VirtProductName} deployment in advance in order to take advantage of disaster recovery.
+
+Metropolitan Disaster Recovery (Metro-DR)::
+Metro-DR provides two-way synchronous data replication between OpenShift Virtualization clusters installed at two separate sites. Use Metro-DR during a site disaster to fail applications to the additional site. This synchronous solution is only available to metropolitan distance data centers with a 10 millisecond latency or less. For more information, see: https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.14/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#metro-dr-solution[Metro-DR solution for OpenShift Data Foundation].
```
Comment (kgoldbla) [inline virt/backup_restore/virt-disaster-recovery.adoc:14]: [1]between OpenShift Virtualization clusters -> between managed OpenShift Virtualization clusters
[2]installed at two separate sites -> installed on the primary and secondary sites.
[3] to fail applications to the additional site - to fail over applications from the primary to the secondary site and later after restoring the disaster site to relocate the application back from the secondary to the primary site.

**Review by kgoldbla — COMMENTED** [review]
lgtm

File: virt/backup_restore/virt-disaster-recovery.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+[id="virt-disaster-recovery"]
+= Disaster recovery
+include::_attributes/common-attributes.adoc[]
+:context: virt-about-dr
+
+toc::[]
+
+The disaster recovery documentation provides information for administrators on
+how to recover from certain disaster scenarios that might occur with an
+{product-title} cluster. As an administrator, you must plan your {VirtProductName} deployment in advance in order to take advantage of disaster recovery.
```
Comment (adellape) [inline virt/backup_restore/virt-disaster-recovery.adoc:11]: Lines should be [unwrapped](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines).

File: virt/backup_restore/virt-disaster-recovery.adoc [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Metropolitan Disaster Recovery (Metro-DR)::
+Metro-DR provides two-way synchronous data replication between managed {VirtProductName} clusters installed on primary and secondary sites. Use Metro-DR during a site disaster to fail applications from the primary to the secondary site, and to relocate the application back to the primary site after restoring the disaster site. This synchronous solution is only available to metropolitan distance data centers with a 10 millisecond latency or less. For more information, see: https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.14/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#metro-dr-solution[Metro-DR solution for OpenShift Data Foundation].
```
Comment (adellape) [inline virt/backup_restore/virt-disaster-recovery.adoc:14]: Colon not needed:
```suggestion
Metro-DR provides two-way synchronous data replication between managed {VirtProductName} clusters installed on primary and secondary sites. Use Metro-DR during a site disaster to fail applications from the primary to the secondary site, and to relocate the application back to the primary site after restoring the disaster site. This synchronous solution is only available to metropolitan distance data centers with a 10 millisecond latency or less. For more information, see https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.14/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#metro-dr-solution[Metro-DR solution for OpenShift Data Foundation].
```

File: virt/backup_restore/virt-disaster-recovery.adoc [hunk 4]
```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-disaster-recovery"]
+= Disaster recovery
```
Comment (adellape) [inline virt/backup_restore/virt-disaster-recovery.adoc:3]: Should this match the "About disaster recovery" title used in left-nav, or vice versa?

**Review by adellape — APPROVED** [review]

**Pull request merged**